bot/constants.py

```diff
@@ -4,11 +4,27 @@
 
 import yaml
 
+
 # env vars
 PREFIX = os.getenv("PREFIX") or "!"
 TOKEN = os.getenv("TOKEN")
 BOT_REPO_URL = "https://github.com/gurkult/gurkbot"
 
+
+class Roles(typing.NamedTuple):
+    announcements = int(os.getenv("ANNOUNCEMENTS_ID", "789978290844598272"))
+    polls = int(os.getenv("POLLS_ID", "790043110360350740"))
+
+
+class Emojis(typing.NamedTuple):
```

Comment:
```suggestion
class Emojis(NamedTuple):
```
Also move this class below `Channels`